### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for items 21001, 21002, 21004, 21006, 21016

https://divine-pride.net/database/item/21001
``````
update script
``````

https://divine-pride.net/database/item/21002
``````
update script
``````

https://divine-pride.net/database/item/21004
``````
remove job Swordman
update script
``````

https://divine-pride.net/database/item/21006
``````
update script
``````

https://divine-pride.net/database/item/21016
``````
update script
``````

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -83532,7 +83547,12 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      .@r = getrefine(); bonus bBaseAtk,(.@r*6); bonus bMatk,(.@r*2); .@i = min(BaseLevel/10,12); if(.@i>2) bonus bBaseAtk,((.@i-2)*5);
+      .@r = getrefine();
+      bonus bBaseAtk,(.@r*6);
+      bonus bMatk,(.@r*2);
+      .@i = min(BaseLevel/10,12);
+      if(.@i>2)
+         bonus bBaseAtk,((.@i-2)*5);
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml:83555]: No change here?

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -83775,6 +83797,7 @@ Body:
     Refineable: true
     Script: |
       bonus bAtk,pow(min(getrefine(),15),2);
+      bonus bMatk,pow(min(getrefine(),15),2);
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml:83800]:
```suggestion
      bonus bMatk,pow(min(getrefine(),15),2)/2;
```

**Review by Lemongrass3110 — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -83443,7 +83443,19 @@ Body:
...
+      bonus2 bAddRace,RC_Player_Doram,80;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Human,30;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Doram,30;
+      .@r = getrefine();
+      if(.@r>=6) {
+         bonus2 bAddRace,RC_Player_Human,40;
+         bonus2 bAddRace,RC_Player_Doram,40;
+      }
+      if(.@r>=9) {
+         bonus bShortWeaponDamageReturn,20;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:83456]:
```suggestion
         bonus bShortWeaponDamageReturn,20;
         bonus bMagicDamageReturn,20;
```
according to aegis db

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -83484,7 +83497,8 @@ Body:
     EquipLevelMin: 48
     Refineable: true
     Script: |
-      bonus bCritical,30; bonus bAspdRate,8; bonus2 bAddEff2,Eff_Curse,10;
+      bonus bCritical,30; bonus bAspdRate,8;
+      bonus2 bAddEff2,Eff_Curse,10;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:83501]: yes

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -83498,12 +83512,13 @@ Body:
     Jobs:
       Crusader: true
       Knight: true
-      Swordman: true
     Locations:
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:83515]: thanks

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:83555]: yes

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:83800]: thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:83456]: thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**